plugins/org.eclipse.n4js.utils/src/org/eclipse/n4js/utils/nodemodel/NodeModelUtilsN4.xtend

```diff
@@ -45,4 +69,27 @@ class NodeModelUtilsN4 {
 		val ge = node.grammarElement;
 		return if (ge instanceof Keyword) keyword.equalsIgnoreCase(ge.value) else false;
 	}
+
+	def public static String getFullTextOfBogusType(INode node) {
```

Comment:
Please add documentation and rename the method. It is independent from the Bogus Type, isn't it? Maybe you can add a reference to the NodeModelUtils method getTokenText, which returns the token text without whitespaces only.